tests/ConfigTest.php

```diff
@@ -208,6 +208,38 @@ public function testSender()
         $c->send($p, $this->token);
     }
     
+    public function testEndpoint()
```

Comment:
Can we add a test that doesn't set the endpoint in the config just to ensure the default is propagating through properly?